Forge/src/main/java/me/ferdz/placeableitems/init/PlaceableItemsBlockRegistry.java

```diff
@@ -396,6 +400,7 @@ public static void onBlocksRegistry(final RegistryEvent.Register<Block> event) {
                 .register("firework_block", Items.FIREWORK_ROCKET);
         GLASS_BOTTLE = new PlaceableItemsBlockBuilder()
                 .addComponent(new BiPositionBlockComponent())
+                .addComponent(new FluidHolderBlockComponent(1000))
```

Comment:
One thing that should be taken into consideration is the strive for realism. Might it be worth including a predicate here to disallow lava to be placed in a glass container? Or is this something that should be permitted for the sake of decoration.

(Fluid whitelisting by predicate is already supported, it's just a matter of whether or not one should be applied to the glass bottle)

Reply:
I feel like it should be allowed, we're striving for decoration & aesthetics, so the more options the users have the better it is, even to the cost of realism